SignaturePad-Example/src/main/java/com/github/gcacace/signaturepad/MainActivity.java

```diff
@@ -106,4 +111,21 @@ public boolean addSignatureToGallery(Bitmap signature) {
         }
         return result;
     }
+
+    public boolean addSignatureToGallery(String signatureSvg) {
```

Comment:
Please rename this function in something like `addSvgSignatureToGallery`, and rename also the other function generating the JPG/PNG image in `addJpgSignatureToGallery`.